src/chunk_insert_state.c

```diff
@@ -238,13 +239,9 @@ chunk_insert_state_create(Chunk *chunk, ChunkDispatch *dispatch, CmdType operati
 	MemoryContext cis_context = AllocSetContextCreate(dispatch->estate->es_query_cxt,
 													  "chunk insert state memory context",
 													  ALLOCSET_DEFAULT_SIZES);
-	Query	   *parse = dispatch->parse;
-	OnConflictAction onconflict = ONCONFLICT_NONE;
+	OnConflictAction onconflict = dispatch->on_conflict;
```

Comment:
Do we need this extra variable now? Can't we just use `dispatch->on_conflict` below?